pages/k8s/release-notes.md

```diff
@@ -14,6 +14,83 @@ toc: False
 ---
 
 <!-- AUTOGENERATE RELEASE NOTES HERE -->
+# 1.26
+
+### December ??, 2022 - `charmed-kubernetes --channel 1.26/stable` 
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.26/bundle.yaml).
```

Comment:
@addyess i can't remember how we make the bundle subdirs; is that a manual copy or does something in the release process do it for us?  I ask because currently this link is 404.

Reply:
The addyess bot does it when we have a bundle. I don't think this is a blocker as long as we remember to actually do it - there is plenty of delay in actually getting the site updated and more than enough time to paste files into the repo